packages/client/src/utils/referenceApi.ts

```diff
@@ -94,6 +106,40 @@ async function loadConfig(): Promise<IApplicationConfig> {
   return response
 }
 
+async function loadCertificatesTemplatesDefinitions(): Promise<
+  ICertificateResponse
+> {
+  const url = `${window.config.CONFIG_API_URL}/getActiveCertificates`
```

Comment:
Should this be making a GraphQL request to `getActiveCertificatesSVG` you've defined [here](https://github.com/opencrvs/opencrvs-core/pull/2442/files#diff-ddf0b51b91fbadc0bbab52f69b1b145cb9d68741207633289577f390143aea90R44) instead of making a direct call to the config service?

Reply:
I think direct calling will be faster as this will be called during login. And also we have used the same approach for loading Zambia resources by calling direct to the resource package.

Reply:
It would be better if the client would make all requests to the underlying microservices through the gateway. That way we wouldn't have to expose the services to the public internet, making the whole system a bit more secure.  It's also a question of mixing plain HTTP requests and GraphQL requests, which isn't ideal either as with plain HTTP we do not get the same type guarantees, validation and type generation we get with GraphQL. I think the GraphQL endpoint you've defined is perfectly fine and you should use it here. Otherwise, it won't be used anywhere, right?

Reply:
Yeah. I got your point. I have modified the code to fetch data from gql.